### ISSUE TITLE
[ONNX] Increase coverage of quantization operators

### ISSUE THREAD

**Issue by garymm** [pytorch/pytorch#80039]
Up-to-date supported list: https://pytorch.org/docs/master/onnx_supported_aten_ops.html?highlight=onnx

------

Extending support for operators used by quantization. A few thoughts/options.

1. Support all operators documented here https://pytorch.org/docs/stable/quantization-support.html?highlight=quantization. The list is non-exhaustive and there may exist undocumented quantized operators.
    * torch
        - [x] quantize_per_tensor
        - [x] quantize_per_channel
        - [x] dequantize

    * other aten
        - [ ] slice

    * torch.Tensor
        - [x] view
        - [x] as_strided
        - [x] expand
        - [x] flatten
        - [x] select
        - [x] ne
        - [x] eq
        - [x] ge
        - [x] le
        - [x] gt
        - [x] lt
        - [ ] copy_
        - [x] clone
        - [ ] dequantize
        - [ ] equal
        - [ ] int_repr
        - [ ] max
        - [x] mean
        - [ ] min
        - [ ] q_scale
        - [ ] q_zero_point
        - [ ] q_per_channel_scales
        - [ ] q_per_channel_zero_points
        - [ ] q_per_channel_axis
        - [ ] ~resize_~ [resize_ is not traceable]
        - [ ] sort
        - [ ] topk

    * torch.nn.quantized
        - [x] ReLU6
        - [x] Hardswish
        - [ ] ELU
        - [x] LeakyReLU
        - [x] Sigmoid
        - [ ] BatchNorm2d
        - [ ] BatchNorm3d
        - [ ] Conv1d
        - [x] Conv2d
        - [ ] Conv3d
        - [ ] ConvTranspose1d
        - [ ] ConvTranspose2d
        - [ ] ConvTranspose3d
        - [x] Embedding
        - [x] EmbeddingBag
        - [ ] FloatFunctional
        - [ ] FXFloatFunctional
        - [ ] QFunctional
        - [x] Linear
        - [x] LayerNorm
        - [x] GroupNorm
        - [x] InstanceNorm1d
        - [x] InstanceNorm2d
        - [x] InstanceNorm3d

    * torch.nn.quantized.functional
        - [ ] avg_pool2d
        - [ ] avg_pool3d
        - [ ] adaptive_avg_pool2d
        - [ ] adaptive_avg_pool3d
        - [ ] conv1d
        - [x] conv2d
        - [ ] conv3d
        - [ ] interpolate
        - [x] linear
        - [ ] max_pool1d
        - [ ] max_pool2d
        - [ ] celu
        - [x] leaky_relu
        - [x] hardtanh
        - [x] hardswish
        - [ ] threshold
        - [x] elu
        - [x] hardsigmoid
        - [ ] clamp
        - [ ] upsample
        - [ ] upsample_bilinear
        - [ ] upsample_nearest
1. Support all existing supported full precision pytorch/vision models with quantization.
1. Automate the discovery (support) for quantized operators.

**Comment by justinchuby** [comment]
Seen:

- Weights and biases: some are quantized, others not

**Comment by abock** [comment]
@justinchuby can you link the other tracking issue here?
